index.html

```diff
@@ -18,7 +18,7 @@
     <div class="container">
       <div class="row">
         <div class="column">
-          <!-- Night Mode Creation -->
+          <!-- Night Mode Creation --
```

Comment:
Oops! Looks like you accidentally deleted the `>` character. 😅 

No worries, these things happen all the time. It's part of why code review exists! 🎉 

If this doesn't get replaced, it breaks the entire document for everyone, as the web browser will now think that everything until the next properly-closed comment is a comment. 😞 

Please make an extra commit to replace this.